components/mamo_business/actions/create-payment-link/create-payment-link.mjs

```diff
@@ -0,0 +1,245 @@
+  type: "action",
+  props: {
+    app,
+    title: {
+      type: "string",
+      label: "Title",
+      description: "The title of the payment link.",
+    },
+    description: {
+      type: "string",
+      label: "Description",
+      description: "Payment description. This will appear on the payment checkout page.",
+      optional: true,
+    },
+    capacity: {
+      type: "integer",
+      label: "Capacity",
+      description: "The capacity will be ignored when the subscription params exist and value will be null.",
+      optional: true,
+    },
+    active: {
+      type: "boolean",
+      label: "Active",
+      description: "Whether the payment is active or not.",
+      optional: true,
+    },
+    returnUrl: {
+      type: "string",
+      label: "Return URL",
+      description: "The URL which the customer will be redirected to after a successful payment.",
+      optional: true,
+    },
+    failureReturnUrl: {
+      type: "string",
+      label: "Failure Return URL",
+      description: "The URL which the customer will be redirected to after a failure payment.",
+      optional: true,
+    },
+    processingFeePercentage: {
+      type: "integer",
+      label: "Processing Fee Percentage",
+      description: "The percentage of the transaction that is the fee.",
+      optional: true,
+    },
+    amount: {
+      type: "string",
+      label: "Amount",
+      description: "The value number of the payment.",
+      optional: true,
+    },
+    amountCurrency: {
+      type: "string",
+      label: "Amount Currency",
+      description: "The currency that the transaction will be charged.",
+      default: "AED",
+      options: [
+        "AED",
+        "USD",
+        "EUR",
+        "GBP",
+        "SAR",
+      ],
+    },
+    isWidget: {
+      type: "boolean",
+      label: "Is Widget",
+      description: "Generate widget payment link.",
+      optional: true,
+    },
+    enableTabby: {
+      type: "boolean",
+      label: "Enable Tabby",
+      description: "Enables the ability for customers to buy now and pay later.",
+      optional: true,
+    },
+    enableMessage: {
+      type: "boolean",
+      label: "Enable Message",
+      description: "Enables the ability for customers to add a message during the checkout process.",
+      optional: true,
+    },
+    enableTips: {
+      type: "boolean",
+      label: "Enable Tips",
+      description: "Enables the tips option. This will be displayed on the first screen.",
+      optional: true,
+    },
+    enableCustomerDetails: {
+      type: "boolean",
+      label: "Enable Customer Details",
+      description: "Enables adding customer details such as the name, email, and phone number. This screen will be displayed before the payment details screen.",
+      optional: true,
+    },
+    enableQuantity: {
+      type: "boolean",
+      label: "Enable Quantity",
+      description: "The number of times a payment link can be used.",
```

Comment:
This description makes the prop seem like an integer, perhaps it can be improved